src/dashboard/src/main/migrations/0034_mediaconch_policy_checks.py

```diff
@@ -0,0 +1,451 @@
+        tasktype=user_choice_type,
+        tasktypepkreference=None,
+        description='Perform policy checks on originals?'
+    )
+
+    # "Perform policy checks on originals?" CHOICE chain link.
+    rgnl_policy_check_choice_cl_pk = \
+        '70fc7040-d4fb-4d19-a0e6-792387ca1006'
+    rgnl_policy_check_choice_cl = MicroServiceChainLink.objects.create(
+        id=rgnl_policy_check_choice_cl_pk,
+        currenttask=rgnl_plcy_chck_choice_tc,
+        defaultnextchainlink=None,
+        microservicegroup='Validation'
+    )
+
+    # MS Chain Choice and Chain that say "Yes, I do want to do policy checks on
+    # originals"
+    chain_yes_rgnl_plcy_chck_msc_pk = \
+        'c611a6ff-dfdb-46d1-b390-f366a6ea6f66'
+    chain_yes_rgnl_plcy_chck_msc = MicroServiceChain.objects.create(
+        id=chain_yes_rgnl_plcy_chck_msc_pk,
+        startinglink=rgnl_policy_check_cl,
+        description='Yes'  # Yes, policy checks on originals please!'
+    )
+
+    choice_yes_rgnl_plcy_chck_mscc_pk = \
+        'c4bdc313-3389-49db-a8f1-a0f1b81a8dd2'
+    MicroServiceChainChoice.objects.create(
+        id=choice_yes_rgnl_plcy_chck_mscc_pk,
+        chainavailable=chain_yes_rgnl_plcy_chck_msc,
+        choiceavailableatlink=rgnl_policy_check_choice_cl
+    )
+
+    # MS Chain Choice and Chain that say "No, I do not want to do policy checks
+    # on originals"
+    chain_no_rgnl_plcy_chck_msc_pk = \
+        '3e891cc4-39d2-4989-a001-5107a009a223'
+    chain_no_rgnl_plcy_chck_msc = MicroServiceChain.objects.create(
+        id=chain_no_rgnl_plcy_chck_msc_pk,
+        startinglink=move_examine_contents_cl,  # must crucially point to "Move
+                                                # to examine contents", i.e.,
+                                                # what "Validate formats" used
+                                                # to point to.
+        description='No'  # No policy checks on originals please!'
+    )
+
+    choice_no_rgnl_plcy_chck_mscc_pk = \
+        '7c7feef9-34e9-4745-a5e9-dfb7ba82eba2'
+    MicroServiceChainChoice.objects.create(
+        id=choice_no_rgnl_plcy_chck_mscc_pk,
+        chainavailable=chain_no_rgnl_plcy_chck_msc,
+        choiceavailableatlink=rgnl_policy_check_choice_cl
+    )
+
+    ###########################################################################
+    # END Micro-service that asks "Do you want to do policy checks on ACCESS
+    # derivatives?"
+    ###########################################################################
+
+    # 'Validate formats' exits to 'Policy checks for originals'
+    # CHOICE cl.
+    validate_formats_cl = MicroServiceChainLink.objects.filter(
+        currenttask__description='Validate formats').first()
+    validate_formats_cl.defaultnextchainlink = rgnl_policy_check_choice_cl
+    MicroServiceChainLinkExitCode.objects\
+        .filter(microservicechainlink=validate_formats_cl)\
+        .update(nextmicroservicechainlink=rgnl_policy_check_choice_cl)
```

Comment:
With this you're leaving behind the old task config, right? Should we delete it?